bee-node/bee-plugin/bee-plugin-dashboard/src/routes.rs

```diff
@@ -123,7 +123,7 @@ pub(crate) fn api_routes(
     allowed_routes
         .and(reverse_proxy_filter(
             "".to_string(),
-            "http://".to_owned() + &rest_api_config.bind_socket_addr().to_string() + "/",
+            format!("http://{}/", rest_api_config.bind_socket_addr()),
```

Comment:
:heart:

Reply:
I had to do it because my compiler reports an error here that doesn't even makes sense :trollface: